src/test/java/fr/adrienbrault/idea/symfony2plugin/tests/config/yaml/YamlReferenceContributorTest.java

```diff
@@ -36,4 +39,153 @@ public void testConstantProvidesReferences() {
             PlatformPatterns.psiElement(Field.class).withName("FOO")
         );
     }
+
+    public void testArgumentsSequenceProvidesReferences() {
+        assertReferenceMatchOnParent(
+            YAMLFileType.YML,
+            "services:\n" +
+            "  app.service.bar:\n" +
```

Comment:
are keys not supported by references provider?

so that we can provide support for this also later:

```
App\\BarService: ~
```

```
App\\BarService:
  arguments:
    $test: "%foobar%"
```

Reply:
Thank you for feedback!

> are keys not supported by references provider?

No. I planned to work on this in separate PR. Work in progress branch: https://github.com/Haehnchen/idea-php-symfony2-plugin/compare/master...adamwojs:class_fqn_ref_provider?expand=1

Case when service name is FQCN is a little bit more tricky. What I would expect as Symfony user is that whenever class/namespace is renamed the service name and it's references are updated. This would be a huge time saver.

Good idea with parameters references :+1: